source/client-side-encryption/client-side-encryption.rst

```diff
@@ -325,7 +325,7 @@ MongoClient Changes
       bypassAutoEncryption: Optional<Boolean>; // Default false.
       extraOptions: Optional<Map<String, Value>>;
       tlsOptions: Optional<Map<String, TLSOptions>>; // Maps KMS provider to TLS options.
-      encryptedFieldConfigMap: Optional<Map<String, Document>>; // Maps namespace to EncryptedFieldConfig.
+      EncryptedFieldsMap: Optional<Map<String, Document>>; // Maps namespace to EncryptedFields.
```

Comment:
Nit: lowercase the first letter for consistency

Reply:
Done.